shell.nix

```diff
@@ -11,6 +11,20 @@ pkgs.mkShell {
     helm-docs
     nixos-shell
     kind
+    python311
+    (poetry.override { python3 = python311; })
+    gcc
+    gnumake
+    btrfs-progs
+    stdenv.cc.cc.lib
   ] ++ pkgs.lib.optional (builtins.getEnv "IN_NIX_SHELL" == "pure") [ docker-client ];
+  shellHook = ''
+    export LD_LIBRARY_PATH="${pkgs.stdenv.cc.cc.lib}/lib"
```

Comment:
are there duplicate cc or is it like this?

Reply:
It was needed :)
(For btrfs progs and their libs)